### PR TITLE
TiFlash resource control and pipeline execution model: update docs link and refine wording

### PR DESCRIPTION
**Pull request by Oreoxmt**

### CODE REVIEW

File: releases/release-7.2.0.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.2/quick-start-with-
...
+    Prior to v7.2.0, each task in the TiFlash engine must individually request thread resources during execution. TiFlash controls the number of tasks to limit thread resource usage and prevent overuse, but this issue could not be completely eliminated. To address this problem, starting from v7.2.0, TiFlash introduces a pipeline execution model. This model centrally manages all thread resources and schedules task execution uniformly, maximizing the utilization of thread resources while avoiding resource overuse. To enable or disable the pipeline execution model, modify the [`tidb_enable_tiflash_pipeline_model`](https://docs.pingcap.com/tidb/v7.2/system-variables/#tidb_enable_tiflash_pipeline_model-new-in-v720) system variable.
```
Comment (Oreoxmt) [inline releases/release-7.2.0.md:59]:
```suggestion
    Prior to v7.2.0, each task in the TiFlash engine must individually request thread resources during execution. TiFlash controls the number of tasks to limit thread resource usage and prevent overuse, but this issue could not be completely eliminated. To address this problem, starting from v7.2.0, TiFlash introduces a pipeline execution model. This model centrally manages all thread resources and schedules task execution uniformly, maximizing the utilization of thread resources while avoiding resource overuse. To enable or disable the pipeline execution model, modify the [`tidb_enable_tiflash_pipeline_model`](https://docs.pingcap.com/tidb/v7.2/system-variables#tidb_enable_tiflash_pipeline_model-new-in-v720) system variable.
```

File: tidb-resource-control.md [hunk 2]
```diff
@@ -407,11 +404,7 @@ You can get more information about runaway queries from the following system tab
 
 2. For TiDB Self-Hosted, you can use the `resource-control.enabled` parameter to control whether to use request scheduling based on resource group quotas. For TiDB Cloud, the value of the `resource-control.enabled` parameter is `true` by default and does not support dynamic modification. If you need to disable it for TiDB Dedicated clusters, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
 
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-To disable TiFlash resource control, set the TiFlash config [`enable_resource_control`](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file) to false.
+3. For TiDB Self-Hosted, you can use the `enable_resource_control` configuration item to control whether to enable TiFlash resource control. For TiDB Cloud, <!--TODO: add the behavior of TiFlash resource control in TiDB Cloud-->.
```
Comment (guo-shaoge) [inline tidb-resource-control.md:407]: What's ` <!--TODO: add the behavior of TiFlash resource control in TiDB Cloud-->.`, is it expected?

Comment (guo-shaoge) [reply tidb-resource-control.md:407]: Other lgtm

Comment (guo-shaoge) [reply tidb-resource-control.md:407]: For TiDB Cloud, the value of the `enable_resource_control` parameter is `true` by default and does not support dynamic modification. If you need to disable it for TiDB Dedicated clusters, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md)

Comment (Oreoxmt) [inline tidb-resource-control.md:407]:
```suggestion
3. For TiDB Self-Hosted, you can use the `enable_resource_control` configuration item to control whether to enable TiFlash resource control. For TiDB Cloud, the value of the `enable_resource_control` parameter is `true` by default and does not support dynamic modification. If you need to disable it for TiDB Dedicated clusters, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

Comment (Oreoxmt) [reply tidb-resource-control.md:407]: @guo-shaoge PTAL

**Review by guo-shaoge — APPROVED** [review]

**Pull request merged**